src/index.js

```diff
@@ -1 +1,47 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+let collection = [];
+const form = document.getElementById('addTodo');
+
+function ShowList(arr) {
+  const listToDo = arr.map((b) => `
+    <ul class="testList1" draggable="true">
+          <li><input type="checkbox" id='${b.id}' value='${b.complete}' class="checkboX">
+          <p>${b.title}</p>
+          </li>
+          <i class=" fa fa-ellipsis-v" id='${b.id}'></i>
+          <i class="fa fa-trash-o" id='${b.id}'></i>          
+      </ul>
+     `).join('');
+  document.getElementById('showListItem').innerHTML = `${listToDo}`;
+}
+
+function addList() {
+  const todo = {
+    id: collection.length,
+    title: document.getElementById('todotitle').value,
+    complete: false,
+  };
+  collection.push(todo);
+  localStorage.setItem('todoList', JSON.stringify(collection));
+  if (collection.length > 0) {
+    ShowList(collection);
+  }
+  form.reset();
+}
+
```

Comment:
- [ ] Please make sure it has an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. You can choose to have more properties or name these properties differently.
```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```